### ISSUE TITLE
"lang-sequel" test fails on "make distcheck" 

### ISSUE THREAD

**Issue by hirooih** [universal-ctags/ctags#3093]
`make distcheck` fails on `lang-sequel`.
FYI `make tmain` passes.

The version of ctags: Current HEAD
commit 23ea3765d15a4cc361a1dfef497617c91c704fbe (HEAD -> master, upstream/master)
Merge: 88cd4981 8d3a6bc8
Author: Masatake YAMATO <yamato@redhat.com>
Date:   Thu Jul 15 07:41:49 2021 +0900

```
$ make distclean
...
$ ./autogen.sh
...
$ ./configure
...
$ make distcheck
...
Testing lang-sequel
------------------------------------------------------------
stdout                                                      passed
stderr                                                      failed (diff: /home/hiroo/GitHub/ctags/universal-ctags-5.9.0/_build/sub/Tmain/lang-sequel.d/stderr-diff.txt)
exit                                                        failed (diff: /home/hiroo/GitHub/ctags/universal-ctags-5.9.0/_build/sub/Tmain/lang-sequel.d/exit-diff.txt)
...
Failed tests
============================================================
lang-sequel/stderr-compare (not committed/cached yet)
lang-sequel/exit-compare (not committed/cached yet)

make[1]: *** [Makefile:6315: tmain] Error 1
make[1]: Leaving directory '/home/hiroo/GitHub/ctags/universal-ctags-5.9.0/_build/sub'
make: *** [Makefile:5477: distcheck] Error 1
$ 
```

**Comment by hirooih** [comment]
I found this during debugging #3054.
`make distcheck* falis with the error I reported on this PR.
I tested with 'master' branch and have the same fail, as I reported on this PR.

I'm using Ubuntu 20.04, Windows 10 WSL2.

#3094 is a similar issue. `make distcheck` passed on `fedora33_distcheck`.  `make distcheck` also fails on the branch on my env.

**Comment by hirooih** [comment]
I've sent fixes for this issue on PR #3096.

**Comment by masatake** [comment]
You can close this:-)

**Comment by hirooih** [comment]
Fixed by #3096. Closed
